db/schema.rb

```diff
@@ -120,18 +120,14 @@
   create_table "house_participations", force: :cascade do |t|
     t.integer "house_id", null: false
     t.date "join_date", null: false
-    t.integer "score", null: false
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
+    t.integer "score", default: 0, null: false
     t.bigint "user_id"
     t.index ["user_id"], name: "index_house_participations_on_user_id"
   end
 
   create_table "houses", force: :cascade do |t|
     t.text "house_name", null: false
     t.date "house_start", null: false
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
   end
 
```

Comment:
so this is kind of a problem
i don't know what your migrations are doing but every model should have the created_at and updated_at attributes